filesystem/config.pan

```diff
@@ -866,295 +935,312 @@ variable DISK_PART_BY_DEV = {
+                if ( !exists(SELF['volume_groups']) ) {
+                    SELF['volume_groups'] = dict();
+                };
+                SELF['volume_groups'][dev_name] = dict("device_list", partitions);
+            };
         };
-        SELF['volume_groups'][dev_name] = dict("device_list", partitions);
-      };
     };
-  };
 
-  SELF;
+    SELF;
 };
 
 # Build a list of logical volumes per volume group.
 # They will be properly ordered at creation time, based on file system
 # creation order.
 variable DISK_LV_BY_VG = {
-  foreach (i;device;DISK_DEVICE_LIST) {
-    params = DISK_VOLUME_PARAMS[device];
-    if ( params['type'] == 'lvm' ) {
-      # Already checked for existence
-      params = DISK_VOLUME_PARAMS[device];
-
-      if ( !exists(params['device'])  ) {
-        error("Logical volume name undefined for '"+device+"'");
-      };
-      if ( exists(params['volgroup'])  ) {
-        vg_name = params['volgroup'];
-      } else {
-        error("No volume group defined for logical volume '"+params['device']+"'");
-      };
-      if ( !exists(SELF[vg_name]) ) {
-        SELF[vg_name] = dict();
-      };
-      options = dict();
-      if ( exists(params['size']) ) {
-        options['size'] = params['size'];
-      } else {
-        error('Size has not been specified for logical volume '+params['device']);
-      };
-      if ( exists(params['raid_type'] ) ) {
-        options['raid_type'] = params['raid_type'];
-      };
-      SELF[vg_name][params['device']] = options;
-    };
-  };
+    foreach (i; device; DISK_DEVICE_LIST) {
+        params = DISK_VOLUME_PARAMS[device];
+        if ( params['type'] == 'lvm' ) {
+            # Already checked for existence
+            params = DISK_VOLUME_PARAMS[device];
 
-  SELF;
+            if ( !exists(params['device'])  ) {
+                error("Logical volume name undefined for '%s'", device);
+            };
+            if ( exists(params['volgroup'])  ) {
+                vg_name = params['volgroup'];
+            } else {
+                error("No volume group defined for logical volume '%s'", params['device']);
+            };
+            if ( !exists(SELF[vg_name]) ) {
+                SELF[vg_name] = dict();
+            };
+            options = dict();
+            if ( exists(params['size']) ) {
+                options['size'] = params['size'];
+            } else {
+                error('Size has not been specified for logical volume %s', params['device']);
+            };
+            if ( exists(params['raid_type'] ) ) {
+                options['raid_type'] = params['raid_type'];
+            };
+            SELF[vg_name][params['device']] = options;
+        };
+    };
+    SELF;
 };
 
 
 "/system/blockdevices/logical_volumes" = {
-  if ( is_defined(DISK_LV_BY_VG) ) {
-    foreach (vg_name;lv_list;DISK_LV_BY_VG) {
-      lvo_add (vg_name, lv_list);
+    if ( is_defined(DISK_LV_BY_VG) ) {
+        foreach (vg_name;lv_list;DISK_LV_BY_VG) {
```

Comment:
panlint is complaining about this one miss.

Reply:
Argh! That was probably one of the lines caught in conflict resolution :disappointed: